projects/commudle-admin/src/app/app-routing.module.ts

```diff
@@ -134,6 +135,10 @@ const routes = [
     path: 'newsletters/:main_newsletter_id',
     component: MainNewsletterComponent,
   },
+  {
```

Comment:
Let's convert the newsletter into a routing module and lazy load it here

Reply:
done